assets/js/src/segments/subscribers_calculator.ts

```diff
@@ -0,0 +1,103 @@
+import MailPoet from 'mailpoet';
+import _ from 'lodash';
+
+export interface FormItem {
```

Comment:
Is it possible to have a better type? The item shouldn't have category_id and `wordpressRole` at the same time, right?

Also, can we rename the items so that we don't combine underscore and camelcase?

Reply:
> The item shouldn't have category_id and wordpressRole at the same time, right?

How can I do it? Could you help me, please?

> Also, can we rename the items so that we don't combine underscore and camelcase?

I used current input names from the form.

Reply:
> I used current input names from the form.

I know and I think we should rename those input names to make them consistent.

Reply:
Could we use something like this?
```
interface FormItem { 
  segmentType: string; 
} 

interface WordpressRoleFormItem extends FormItem { 
   wordpressRole?: string;
} 

interface WooCommerceFormItem extends FormItem { 
 ...
}

type AnyFormItem = WordpressRoleFormItem | WooCommerceFormItem

let lastFormItem: AnyFormItem;
```